components/stimulation/StimulationStudioBlockViewEditor.vue

```diff
@@ -214,9 +214,8 @@ export default {
         this.rest_duration = JSON.stringify(this.get_rest_duration);
         this.stimulation_type_idx = +(this.stimulation_type === "V");
 
-        const stim_until_complete = this.stop_setting === "Stimulate Until Complete";
-        this.stop_option_idx = +stim_until_complete;
-        this.disabled_time = stim_until_complete;
+        this.stop_option_idx = this.run_until_stopped ? 0 : 1;
```

Comment:
nitpick, you can just do `Number(!this.run_until_stopped)` or even `+!this.run_until_stopped`